src/lib/connector.ts

```diff
@@ -5,7 +5,11 @@ import { read, write } from "./query.js";
 import { create } from "./create.js";
 import { hash } from "./hash.js";
 import { siwe } from "./siwe.js";
-import { receipt } from "./tableland-calls.js";
+import {
+  receipt,
+  setController,
```

Comment:
This works well for testnets, but when we go to mainnets we will want a way to go direct to the SC.
If we create a file with a wrapper method for this, the wrapper method could determine if we want to go direct to the SC or relay through the Validator based on the SUPPORTED_CHAIN that was specified. 
Thoughts?

Reply:
I think that would likely be fine. In that case, would be API remain essentially the same? If yes, I'd like to get this in sooner rather than later, unless we think we can push the direct to SC stuff out soon?

Reply:
The direct to SC stuff was merged to main in #71 
I was thinking we would do something similar here